libz-rs-sys-cdylib/Cargo.toml

```diff
@@ -28,9 +28,14 @@ rust-allocator = ["libz-rs-sys/rust-allocator", "libz-rs-sys/std"] # by default,
 custom-prefix = ["libz-rs-sys/custom-prefix"] # use the LIBZ_RS_SYS_PREFIX to prefix all exported symbols
 semver-prefix = ["libz-rs-sys/semver-prefix"] # prefix all symbols in a semver-compatible way
 capi = []
+gz = ["dep:libc"] # gzip support has to be configured explicitly, to avoid a libc dependency in the core zlib-rs
+export-symbols = [] # whether the zlib api symbols are publicly exported
```

Comment:
Nothing seems to be using this feature.

Reply:
yeah, the point of this crate is to expose the symbols, so we don't need that feature flag here.

Reply:
Do we still need the associated function annotations in `gz.rs`?
```
#[cfg_attr(feature = "export-symbols", export_name = crate::prefix!(gzopen))]
pub unsafe extern "C-unwind" fn gzopen(path: *const c_char, mode: *const c_char) -> gzFile {
```
etc?

Reply:
Those can be just `#[export_name = crate::prefix!(gzopen)]` etc. for now